### ISSUE TITLE
Standings

### ISSUE THREAD

**Issue by interludic** [dolejska-daniel/challonge-api#1]
Hello,

do you have any examples on how standings look?

i cant seem to find example of this in your code.

**Comment by dolejska-daniel** [comment]
Hello, @interludic!

`Participant` object has a property `final_rank` in their data list (_it is not yet in an object property_) - thanks to this property, you can find out, what their rank after finalizing the tournament is.

You can get to it by requesting the participant list, then getting the single participant (first participant in the list __won't have to be the first in the ranking__) and now you can get their rank.
```php
$list = pList(); // requesting the participant list
$p = reset($list->getParticipants()); // fetch the first participant for example
$p->getData()['final_rank']; // getting their final rank
```

However, you are right that __it is not implemented yet__, so __I will try to make a commit today, to implement this feature__ and make working with standing a bit easier :). And I also noticed a few minor bugs, which I shall fix!

Thanks for the feedback 👍!

**Comment by dolejska-daniel** [comment]
So - since it was nothing difficult, I've made the commit now. You can use the new feature by calling `$api->tStandings(...)`. The API will return `ParticipantList` with participants ordered by their `final_rank` in descendant order.

Implemented in 836f7d16c783b0a26f158fd2311b61e59c8fc288.
